companies/studydrive.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "studydrive",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "school"
+    ],
+    "name": "Studydrive GmbH",
+    "address": "Mehringdamm 32\n10961 Berlin\nDeutschland",
+    "phone": "+49 (030) 555 794 64",
+    "email": "info@studydrive.net",
+    "web": "https://www.studydrive.net",
+    "sources": [
+        "https://www.studydrive.net/datenschutz",
+        "https://www.studydrive.net/impressum"
+    ],
+    "required-elements": [
```

Comment:
We had agreed to always include the name element.

*You* actually insisted on that. :P

Reply:
Well…